ArcGISRuntimeSDKQt_CppSamples/Maps/DownloadPreplannedMap/DownloadPreplannedMap.cpp

```diff
@@ -217,10 +183,61 @@ void DownloadPreplannedMap::loadPreplannedMapAreas()
         return;
       }
 
-      Graphic* areaOfInterest = new Graphic(mapArea->areaOfInterest(), m_graphicParent.get());
+      Graphic* areaOfInterest = new Graphic(mapArea->areaOfInterest(), this);
       m_graphicsOverlay->graphics()->append(areaOfInterest);
     });
 
     mapArea->load();
   }
 }
+
+void DownloadPreplannedMap::downloadPreplannedMapArea(const QString& path, int index)
+{
+  connect(m_offlineMapTask, &OfflineMapTask::createDefaultDownloadPreplannedOfflineMapParametersCompleted, this, [this, path] (QUuid ,const DownloadPreplannedOfflineMapParameters& parameters)
+  {
+    m_params = parameters;
+
+    /* Set the update mode to not receive updates.
+     * Other options:
+     * SyncWithFeatureServices - changes will be synced directly with the
+     * underlying feature services.
+     * DownloadScheduledUpdates - schedulded, read-only updates will be
+     * downloaded and applied to the local geodatabase. */
+    m_params.setUpdateMode(PreplannedUpdateMode::NoUpdates);
+
+    m_preplannedMapJob = m_offlineMapTask->downloadPreplannedOfflineMap(m_params, path);
```

Comment:
I don't see anywhere that you are handling progress from the job. That's fine but I think the README suggests that progress will be displayed?

Reply:
Ah good catch! I can add that in easily.